static/js/tools/map/chart_loader.tsx

```diff
@@ -145,29 +157,47 @@ function loadChartData(
   populationData: { [dcid: string]: SourceSeries },
   isPerCapita: boolean,
   geoJsonData: GeoJsonData,
+  placeInfo: PlaceInfo,
   setChartData: (data: ChartData) => void
 ): void {
-  const dataValues = {};
+  const mapDataValues = {};
+  const breadcrumbDataValues = {};
   const sources: Set<string> = new Set();
   const statVarDates = {};
   for (const dcid in statVarData.stat) {
     if (_.isEmpty(statVarData.stat[dcid])) {
       continue;
     }
-    const statVarValue = statVarData.stat[dcid].value;
-    const importName = statVarData.stat[dcid].metadata.importName;
-    sources.add(statVarData.metadata[importName].provenanceUrl);
-    statVarDates[dcid] = statVarData.stat[dcid].date;
+    let statVarValue = statVarData.stat[dcid].value;
     if (isPerCapita && dcid in populationData) {
       const popDate = getPopulationDate(
         populationData[dcid],
         statVarData.stat[dcid]
       );
-      dataValues[dcid] = statVarValue / populationData[dcid].data[popDate];
+      statVarValue = statVarValue / populationData[dcid].data[popDate];
       sources.add(populationData[dcid].provenanceUrl);
-    } else if (!isPerCapita) {
-      dataValues[dcid] = statVarValue;
+    } else if (isPerCapita && !(dcid in populationData)) {
+      continue;
```

Comment:
this case should be handled and shown as a data unavailable (though the hover can still show the unadjusted values)

Reply:
when drawing the map, if there are no values for a certain place dcid, it will show as "data missing". I think it would be confusing for the user if the tooltip showed the unadjusted values